config/locales/catarse_bootstrap/activerecord.pt.yml

```diff
@@ -188,7 +188,7 @@ pt:
             online_days:
               greater_than: "O prazo para arrecadação deve ser maior que 0 dias"
               not_a_number: "O prazo para arrecadação deve ser um número de dias"
-              less_than_or_equal_to: "O prazo para arrecadação deve ser menor que 60 dias"
+              less_than_or_equal_to: "devem ser menores que 60 dias"
```

Comment:
Acho que aqui ficaria melhor `deve ser menor que 60`. Já que na frase ficaria: `Dias online deve ser menor que 60`.

Falando nisso, se o nome for `Quantidade de dias online` acho que ficaria melhor 💅